sympy/physics/control/lti.py

```diff
@@ -76,6 +77,54 @@ def bilinear(tf, sample_per):
         return num_coefs, den_coefs
 
 
+def backward_diff(tf, sample_per):
+
+        """
+
+        Returns falling coefficients of H(z) from numerator and denominator.
+        Where H(z) is the corresponding discretized transfer function,
+        discretized with the backward difference transform method.
+        H(z) is obtained from the continuous transfer function H(s)
+        by substituting s(z) =  (z-1)/(T*z) into H(s), where T is the
+        sample period.
+        H(z) corresponds to the difference equation of the by backward difference scheme.
+        Coefficients are falling, i.e. H(z) = (az+b)/(cz+d) is returned
+        as [a, b], [c, d].
+
+        Examples
+
+        ========
+
+        >>> from sympy.physics.control.lti import TransferFunction, backward_diff
+        >>> from sympy.abc import s, L, R, T
+        >>> tf = TransferFunction(1, s*L + R, s)
+        >>> numZ, denZ = backward_diff(tf, T)
+        >>> numZ
+        [T, 0]
+        >>> denZ
+        [L + R*T, -L]
+        """
+
+        z = Symbol('z') # discrete variable z
+        T = sample_per  # and sample period T
+        s = tf.var
+
+        np = tf.num.as_poly(s).all_coeffs()
+        dp = tf.den.as_poly(s).all_coeffs()
+
+        # The next line results from multiplying H(z) with z^N/z^N
+
+        N = max(len(np), len(dp)) - 1
+        num = Add(*[ T**(N-i)*c*(z-1)**i*(z)**(N-i) for c, i in zip(np[::-1], range(len(np))) ])
+        den = Add(*[ T**(N-i)*c*(z-1)**i*(z)**(N-i) for c, i in zip(dp[::-1], range(len(dp))) ])
+
+        num_coefs = num.as_poly(z).all_coeffs()
+        den_coefs = den.as_poly(z).all_coeffs()
+
+
+        return num_coefs, den_coefs
```

Comment:
The function shouldn't be doubly indented.

Reply:
👍🏽